webui/react/src/pages/WebhookList.tsx

```diff
@@ -160,11 +176,24 @@ const WebhooksView: React.FC = () => {
   );
 
   const columns = useMemo(() => {
-    const actionRenderer = (_: string, record: Webhook) => (
-      <Dropdown menu={DROPDOWN_MENU} onClick={(key) => handleDropdown(key, record)}>
-        <Button icon={<Icon name="overflow-vertical" title="Action menu" />} type="text" />
-      </Dropdown>
-    );
+    const renderMenu: (record: Webhook) => MenuItem[] = (record: Webhook) => {
+      const DROPDOWN_MENU = [];
+      record.name && DROPDOWN_MENU.push({ key: MenuKey.CopyName, label: 'Copy Webhook Name' });
+      canEditWebhooks(workspaces, record) &&
+        DROPDOWN_MENU.push(
+          { key: MenuKey.TestWebhook, label: 'Test Webhook' },
```

Comment:
does testing the webhook make sense to hide behind the edit check?

Reply:
My thought is that we usually only test right after the creation of webhook, because the user needs to have access to the webhook URL for the testing to make sense. When the webhook is in use, testing it should be a rare scenario.